FirebaseAuth/Tests/Unit/FIRAuthAPNSTokenTests.m

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+#include <TargetConditionals.h>
+#if !TARGET_OS_OSX
```

Comment:
Is there a reason why some tests use `TARGET_OS_IOS` and some `!TARGET_OS_OSX`?

Reply:
Only iOS versus iOS and tvOS

Reply:
Maybe worth commenting for the future. Though I'm fine with leaving it as is.

Reply:
Good point. CocoaPods manages this by explicit excludes in the podspec. A better long term solution might be to organize the unit tests directories corresponding to the platforms on which they're supported. cc: @rosalyntan